src/main/drivers/dshot.c

```diff
@@ -139,6 +141,17 @@ uint16_t getDshotTelemetry(uint8_t index)
     return dshotTelemetryState.motorState[index].telemetryValue;
 }
 
+uint16_t erpmToRpm(int erpm)
```

Comment:
```suggestion
uint16_t erpmToRpm(uint16_t erpm)
```

`erpm` is sometimes `int`, sometimes `int16_t` or `uint16_t`. What we really need here?

Reply:
Reworked